content/articles/where-incorporate-eu-citizen.md

```diff
@@ -0,0 +1,162 @@
+
+<blockquote class="twitter-tweet" data-lang="en-gb"><p lang="en" dir="ltr"><a href="https://twitter.com/WeAreWizardsIO">@WeAreWizardsIO</a> No timeline as of yet, sorry! Feel free to sign up here and we&#39;ll notify you as soon as we&#39;re ready <a href="https://t.co/vbuzSSxceJ">https://t.co/vbuzSSxceJ</a></p>&mdash; Stripe (@stripe) <a href="https://twitter.com/stripe/status/757525171943120896">25 July 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+#### Taxes
+Estonia has a very competitive tax system. The corporation tax is **0%** as long as the money stays into the company bank account. Once you take dividends, those are taxed at a flat **20%** rate.
+
+TODO: email from julius was a bit unclear about the 0%
+> But I would mention that there is always a taxation issue with the company, as this is one of the most significant topics for entrepreneurs. Usually you pay taxes according to this legislation where the value is created, so if You would like to establish an Estonian company then I would definitely suggest You to contact tax advisor who could help You getting a better understanding of taxation details, for example if you would like to “use” the Estonian 0% corporate income tax, then how much value you have to create here to make it applicable.
+
+#### How to incorporate/open bank account
+Estonia is at the forefront of e-residency and therefore don't need to go the country to get it or incorporate. You currently need to go to Estonia to open a bank account but starting September it will apparently be possible to open one through Skype.
+
+[Leapin](https://www.leapin.eu) is recommended by the e-residency website and offers incorporation and accounting for less than 100€ per month. It only supports a single shareholder apparently though.
+
+It can take a while to get your e-residency card and you need to pick it up yourself, which might mean a trip to Estonia if there isn't an Estonian embassy around you. Incorporation takes about a day.
+
+### Singapore
+
+#### Trivia
+Famous for its ban on chewing-gum and death penalty, this city state is the only country on that list that isn't in the EU.
+
+It takes the 1st place on the [Doing Business ranking](http://www.doingbusiness.org/rankings) and, more sadly, the 154th position on the [World Press Freedom index](https://rsf.org/en/ranking). It also probably tops the ranking of mall per inhabitant from my own experience as a tourist there.
+
+The immigration process seems pretty open and you should be able to relocate to Singapore without too many issues should you want to do that. I liked Singapore as a tourist but I couldn't say whether it would be a nice place to live or not.
+
+Since Singapore isn't in the EU, you also don't have to deal with VAT MOSS or anything else the EU might think of since it doesn't have jurisdiction over Singapore. Obviously, I'm not a lawyer so don't take my words for it.
```

Comment:
I think the saying is "don't take my word for it" not sure if "words" (plural) works too